.github/workflows/main.yml

```diff
@@ -10,18 +12,89 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    env:
+      ms_suite: omero-ms-zarr-suite
+      bf_suite: ome-zarr-bf2raw-suite
+      zarr_suite: ome-zarr-py-suite
     steps:
-
       - uses: actions/checkout@v2
-
       - name: Setup miniconda
         uses: conda-incubator/setup-miniconda@v2
         with:
           auto-update-conda: true
           channels: conda-forge,ome
           environment-file: environment.yml
           python-version: 3.9
+      - name: Set testing environment
+        if: github.event.action == 'run_test_suite'
+        shell: bash
+        run: |
+          # Set the parameters to be used in the response
```

Comment:
General heads up that this is again quite a block of code to have in yaml.

Reply:
it is. I could look into improving that in a second phase